solutions/bronze/usaco-761.mdx

```diff
@@ -162,10 +158,10 @@ public class Measurement {
 <PySection>
 
 ```py
-NAMES = ['Bessie', 'Elsie', 'Mildred']
+NAMES = ["Bessie", "Elsie", "Mildred"]
```

Comment:
so we're using double quotes for python now, huh?

Reply:
yes

Reply:
yes